Source/BugsnagConfiguration.m

```diff
@@ -231,4 +246,9 @@ - (NSDictionary *)sessionApiHeaders {
              kHeaderApiSentAt: [BSG_RFC3339DateTool stringFromDate:[NSDate new]]
              };
 }
+
+- (BOOL)hasValidApiKey {
+    return _apiKey != nil && ![@"" isEqualToString:_apiKey];
```

Comment:
could we just use `return [_apiKey length] == 0;`?

Reply:
Flipped the boolean but yeah, `[_apiKey length] > 0` is the ticket here.